pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
+line-lenghth = 100
```

Comment:
Let's change it once we've applied everything, if I do it now it will be hard to tease apart what is changing because of the line length, and what still needs formatting from the original unformatted code base 😅

Reply:
ok, anyway this will be the last PR to be merged, so we report for later